### DISCUSSION TITLE
Weight size error

### DISCUSSION THREAD

**Discussion by zhi-yi-huang** [PaddlePaddle/PaddleSpatial#121] [BaiduKDDCup2022] [answered]
When I submit, it says, “The cudnn rnn and setting weight size should be the same”. But I've evaluated it on my local environment, and I was able to complete the evaluations.

**Comment by ramber1836** [comment, marked as answer]
A more detailed error msg is to be returned. Please resubmit and try it again. Hope it works.

**Reply by zhi-yi-huang** [reply]
I have been evaluating for almost 3 hours after I submitted, is this normal?

**Reply by ramber1836** [reply]
It depends. 
You can test your code with the latest version of the toy data (zipped format) on your local machine to estimate the running time.

**Reply by zhi-yi-huang** [reply]
I have tested it is less than 20 minutes.

**Reply by zhi-yi-huang** [reply]
> It depends. You can test your code with the latest version of the toy data (zipped format) on your local machine to estimate the running time.

It's return '异常' now.

**Reply by ramber1836** [reply]
Probably OUT-OF-TIME.
